src/async_fuse/memfs/metadata.rs

```diff
@@ -43,17 +43,6 @@ pub(crate) mod error {
             context: vec![format!("{fn_name}() found fs is inconsistent: {context}",)],
         }
     }
-
-    /// A helper function to build [`DatenLordError::UnsupportedINodeType`].
-    pub(crate) fn build_unsupported_inode_type(
-        node_type: SFlag,
-        context: String,
-    ) -> DatenLordError {
-        DatenLordError::UnsupportedINodeType {
-            node_type,
-            context: vec![context],
-        }
-    }
```

Comment:
`DatenLordError::UnsupportedINodeType`  can be also removed, because it's used no more.